tests/dataSources_test.go

```diff
@@ -737,6 +737,7 @@ var _ = Describe("DataSources", func() {
 			pullMethod        = cdiv1beta1.RegistryPullNode
 			commonAnnotations = map[string]string{
 				"cdi.kubevirt.io/storage.bind.immediate.requested": "true",
+				"cdi.kubevirt.io/storage.deleteAfterCompletion":    "false",
```

Comment:
maybe include a note here that we can get rid of this once we fix https://bugzilla.redhat.com/show_bug.cgi?id=2130509?